client/auth/login.jsx

```diff
@@ -179,7 +179,7 @@ module.exports = React.createClass( {
 				</a>
 				<div className="auth__links">
 					<a href="#" onClick={ this.toggleSelfHostedInstructions }>{ this.translate( 'Add self-hosted site' ) }</a>
-					<a href={ 'https://wordpress.com' + config( 'signup_url' ) } target="_blank">{ this.translate( 'Create account' ) }</a>
+					<a href={ config( 'signup_url' ) + '/desktop' }>{ this.translate( 'Create account' ) }</a>
```

Comment:
This config value for `desktop` is `"signup_url": "/start/desktop",` so we can just remove the `/desktop` here?